x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/security/authz/privilege/IndexPrivilege.java

```diff
@@ -136,7 +136,7 @@ public final class IndexPrivilege extends Privilege {
     );
     private static final Automaton MANAGE_LEADER_INDEX_AUTOMATON = patterns(ForgetFollowerAction.NAME + "*");
     private static final Automaton MANAGE_ILM_AUTOMATON = patterns("indices:admin/ilm/*");
-    private static final Automaton MANAGE_DLM_AUTOMATON = patterns("indices:admin/dlm/*");
+    private static final Automaton MANAGE_LIFECYCLE_AUTOMATON = patterns("indices:admin/dlm/*");
```

Comment:
Sanity check: there are a lot of mentions of `dlm` still, e.g., the action names but also in test code. We are not renaming these here since they are not user-facing and will follow this up with more PRs later, right?